### PR TITLE
update charter

### PR DESCRIPTION
**Pull request by Trott**
Move shared responsibilities with TSC to a sepcial section.

Remove Inclusivity WG as CommComm superseded it. There is currently no
Inclusivity WG nor an initiative to create one.

Minor formatting updates.

This will need to be approved by the board. They will need to approve the corresponding TSC charter changes at the same time.

### CODE REVIEW

**Comment by Trott** [comment]
Corresponding TSC charter changes: https://github.com/nodejs/TSC/pull/569

File: Community-Committee-Charter.md [hunk 1]
```diff
@@ -42,15 +42,27 @@ as a Guest.
...
+("TSC") for:
+
+* Management of the `nodejs` GitHub organization
+* Project governance and process
+* Conduct guidelines
+
+CommComm and TSC will use the `nodejs/admin` GitHub repository to manage shared
+responsibilities. In the event the CommComm and TSC are unable to agree on a
+decision, the result is _status quo_. Such issues may subsequently be escalated
+to the Board for a final decision.
```
Comment (mcollina) [inline Community-Committee-Charter.md]: I'm -1 on giving any power to the Board on our internal governance. The TSC charter was written in this specific way to grant the project the largest autonomy. We should not relinquish that.

https://github.com/nodejs/TSC/pull/569#discussion_r203365775

**Comment by Trott** [comment]
@mcollina I've removed the sentence about escalation to the board. PTAL

**Comment by Trott** [comment]
(Applied the do-not-merge label to make it clear that this needs Board approval before landing. It will need to be approved at the same time as the similar TSC Charter changes.)

**Comment by Trott** [comment]
@bnb @hackygolucky Any chance I can get your reviews on these changes?

**Comment by Trott** [comment]
link to TSC responsibilities added as "See also"

**Comment by bnb** [comment]
Apologies @Trott, will look in a bit 👍

**Comment by Trott** [comment]
Assuming the analogous changes in the TSC Charter are ready to go (which they aren't yet, but we're really close), is this ready to move to the board? Or is there something else that needs to happen first?

**Comment by bnb** [comment]
@Trott I would follow up with @hackygolucky on that 👍

**Comment by Trott** [comment]
This requires https://github.com/nodejs/TSC/pull/569. I'm going to close this because I closed that one for the reasons described in https://github.com/nodejs/TSC/pull/569#issuecomment-421480628. Feel free to re-open, comment, etc. if anyone plans on pushing this forward.

**Pull request closed without merging**